### PR TITLE
MB-6437 Add script/data to benchmark doc viewers

### PR DESCRIPTION
**Pull request by monfresh**
## Description

We need to ensure that the document viewer is performant & reliable in
the transportation offices, which are known to have bandwidth issues.
This PR adds a script to populate a single move with any supported files
from the `/pkg/testdatagen/testdata/bandwidth_test_docs` directory. Supported
file types are PNG, JPG, and PDF. Unsupported files will display an error in the console
when running the script, but should not prevent the script from continuing to run. 
This PR includes 3 files of varying sizes, that both the TOO and TIO can view in the 
orders document viewer and proof of service document viewer respectively.

## Reviewer Notes

To run the script:

```
make db_bandwidth_up
```

To verify the docs were created and can be viewed, follow these steps:

1. Start the server and client:

```
make server_run
make office_client_run
```

2. Click on "Local Sign in" when the office app appears
3. Sign in with the single office user that the script created, and
view the docs as a TOO, and then as a TIO.

As a TOO:

1. Click on the only move in your queue
2. Click on "Orders" in the left sidebar
3. Click on "View & edit orders" in the top right

As a TIO:

1. Click on "Change user role" in the top left
2. Click on "Select transportation_invoicing_officer"
1. Click on the only payment request in your queue
2. Click on "Review service items"

Other things to test:
1. Add more files, or replace the files in the `bandwidth_test_docs` directory with files of your choosing.
2.  Verify that all supported file types are uploaded, but that no unsupported files types are uploaded.

Is there anything you would like reviewers to give additional scrutiny?

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely) have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-6437) for this change

### CODE REVIEW

**Review by jenniferlynparsons — APPROVED** [review]
This is cool! Works as expected.

File: Makefile [hunk 1]
```diff
@@ -702,6 +702,15 @@ db_e2e_up: bin/generate-test-data ## Truncate Test DB and Generate e2e (end-to-e
 	@echo "Populate the ${DB_NAME_TEST} database..."
 	DB_PORT=$(DB_PORT_TEST) go run github.com/transcom/mymove/cmd/generate-test-data --named-scenario="e2e_basic" --db-env="test"
 
+.PHONY: db_bandwidth_up
+db_bandwidth_up: bin/generate-test-data	 ## Truncate Dev DB and Generate data for bandwidth tests
+	@echo "Ensure that you're running the correct APPLICATION..."
+	./scripts/ensure-application app
+	@echo "Truncate the ${DB_NAME_DEV} database..."
+	psql postgres://postgres:$(PGPASSWORD)@localhost:$(DB_PORT_DEV)/$(DB_NAME_DEV)?sslmode=disable -c 'TRUNCATE users CASCADE; TRUNCATE uploads CASCADE;'
```
Comment (duncan-truss) [inline Makefile:710]: Why did you choose the development database over the test database here?  I'm thinking we might end up mimicking how `make e2e_test` works.

Comment (monfresh) [reply Makefile:710]: Mainly for ease of testing my script. We can certainly change it to test if that works better for the Puppeteer script.

**Review by Ryan-Koch — APPROVED** [review]

**Comment by duncan-truss** [comment]
@monfresh I'm worried this doesn't work for uploading to s3.  I tried changing my STORAGE_BACKEND env variable but it looks like we're ignoring that and always using the local storage:

https://github.com/transcom/mymove/blob/58270b71cdd5e9e426ddf7c002345552f0c8d04e/cmd/generate-test-data/main.go#L164-L167

**Comment by duncan-truss** [comment]
I just tested it and it looks like the network throttling will still work with local files, so hopefully my above comment is not a blocker but a nice to have.  I think we'll want the ability to test in a deployed environment though not just locally for other reasons too.

**Comment by monfresh** [comment]
@duncan-truss That's a great point about being able to change the storage system with the `STORAGE_BACKEND` env var. Unfortunately, it's not as easy as using that line from `serve.go`. From what I can tell, the `storer` that is defined in `/cmd/generate-test-data/main.go` is not even used in any of the scenarios (like `e2ebasic` and `devseed`). I also don't see any mention of the storer in our various testdatagen files for creating uploads. I need to dig into the code to see where (or if) the env var is used.

**Comment by monfresh** [comment]
Ah! I see it now. It's passed into the `userUploader` and `primeUploader`, which are used by the testdatagen scenarios and `Make` functions. I'll create a new ticket for this and fix it on Monday.

**Pull request merged**